examples/ios/Examples/QuickStartFlexSync.swift

```diff
@@ -0,0 +1,162 @@
+// :replace-start: {
+//   "terms": {
+//     "QuickStartFlexSync_": ""
+//   }
+// }
+// swiftlint:disable identifier_name
+
+import XCTest
+// :snippet-start: import-realm
+import RealmSwift
+// :snippet-end:
+
+let FS_APP_ID = "swift-flexible-vkljj"
+
+// :snippet-start: model
+class QuickStartFlexSync_Todo: Object {
+    @Persisted(primaryKey: true) var _id: ObjectId
+    @Persisted var name: String = ""
+    @Persisted var status: String = ""
+    @Persisted var ownerId: String
```

Comment:
```suggestion
    @Persisted var owner_id: String
```

I realize it doesn't have to match the template app/tutorial, so this suggestion isn't required for LGTM, but it would be more consistent if you did match the template app/tutorial https://github.com/mongodb-university/realm-template-apps/blob/8562c2d45213b97e3e218795cab8888828777bdc/generated/swiftui-flexible-sync/App/Item.swift